scripts/manage_case

```diff
@@ -1,375 +1,172 @@
-	my $name  = $node->getAttribute('name');
-	my $alias = $node->getAttribute('alias');
-	if ($alias) { 
-	    print " component grid: $name (alias: $alias) \n";
-	} else {
-	    print " component grid: $name \n";
-	}	    
-	my $support; my $nx; my $ny; my $desc;
-	foreach my $child_node ($node->findnodes(".//*")) {
-	    my $name  = $child_node->nodeName();
-	    my $value = $child_node->textContent();
-	    if ($name eq 'support') {$support= $value;}
-	    if ($name eq 'nx') {$nx = $value;}
-	    if ($name eq 'ny') {$ny = $value;}
-	    if ($name eq 'desc') {$desc = $value;}
-	}
-	if (defined $desc)    {print "   $desc \n";}
-	if (defined $nx  )    {print "   nx: $nx ny: $ny \n";}
-	if (defined $support) {print "   support_level: support_level \n";}
-	print " \n";
-    }
-
-    print "\n";
-    print " ======================================================\n";
-    print "                Model Grids:                           \n";
-    print " ======================================================\n";
-    print "\n";
-    foreach my $node ($xml_grid->findnodes(".//grids/grid")) {
-	my @lname_nodes = $node->findnodes("./lname");
-	my $lname = $lname_nodes[0]->textContent();
-	my @alias_nodes = $node->findnodes("./alias");
-	my $alias;
-	if ($#alias_nodes == 0) {
-	    $alias = $alias_nodes[0]->textContent();
-	}
-	my $compset   = $node->getAttribute('compset');
-	my $support   = $node->getAttribute('support');
-
-	print " model grid: $lname \n";
-	if (defined ($alias)  ) { print "    alias        : $alias \n"}
-	if (defined ($compset)) { print "    compset match: $compset \n"}
-	if (defined ($support)) { print "    support level: $support \n";}
-	print " \n";
-    }    
-    print " \n";
-}
-#-------------------------------------------------------------------------------
-sub _queryMachines
-{
-    # Print all currently supported valid grids
-    my ($model, $cimeroot, $xml_config) = @_;
-
-    my @files = $xml_config->findnodes(".//entry[\@id=\"MACHINES_SPEC_FILE\"]/default_value");
-    if ($#files == -1) {
-	die " ERROR: in parsing $file_config for the MACHIENS_SPEC_FILE node \n";
-    } 
-    my $file = $files[0]->textContent();
-    $file =~ s/\$CIMEROOT/$cimeroot/;
-    $file =~ s/\$MODEL/$model/;
-    (-f "$file")  or  die "** Cannot find machines specification file $file ***\n";
-
-    my $model = uc $model;
-    print "\n";
-    print " ======================================================\n";
-    print "            $model Machines:                          \n";
-    print " ======================================================\n";
-    print "\n";
-
-    my $xml_machine = XML::LibXML->new( no_blanks => 1)->parse_file($file);
-    foreach my $machine ($xml_machine->findnodes(".//machine")) {
-	my $name = $machine->getAttribute('MACH');
-	foreach my $node ($machine->findnodes("./DESC")) {
-	    my $desc = $node->textContent();
-	    print " $name :  \n      $desc \n";
-	}
-    }
-}
+#!/usr/bin/env python2.7
```

Comment:
Yes - that is why the comment. I could not get it to work on my mac without this. Should I put in another pull request with this fix?